assets/services.json

```diff
@@ -276,6 +276,15 @@
+      "icon_svg": "<svg xmlns=\"http://www.w3.org/2000/svg\" fill=\"currentColor\" viewBox=\"0 0 92 92\"><path fill=\"#000\" d=\"M32.22 41.78c.75 2.48 1.65 4.8 2.63 7.05a61.84 61.84 0 0 0 4.41 8.79c1.93 3.15 4.06 6.74 7.56 9.06a15.89 15.89 0 0 0 8.66 2.56 15.1 15.1 0 0 0 10.16-4.02c2.28-2.13 5.24-6.23 5.24-14.22 0-2.76 1.22-5.13 2.12-7.02 0-.04.04-.11.12-.27a7.2 7.2 0 0 0 .79-2.84c0-.75-.2-1.61-.71-2.36-.79-1.34-2.2-1.81-3.5-1.81-3.04 0-5.12 2.12-6.19 4.3l-.35.7c-.43-.63-.87-1.3-1.22-1.89a25.33 25.33 0 0 1-3.3-8.9c-.68-3.87-1.78-5.13-3.6-5.6a5.25 5.25 0 0 0-4.05.67 3.95 3.95 0 0 0-3.9-2.56c-2.51 0-4.8 1.65-4.8 4.45 0 .36.04.71.08 1.15a4.29 4.29 0 0 0-1.02-.12c-2.71 0-4.72 1.89-4.72 4.45 0 .59.07 1.26.27 2.05-.24-.04-.47-.04-.7-.04a4.61 4.61 0 0 0-3.43 1.65c-.87 1.03-1.22 2.6-.55 4.77z\"/><path fill=\"#fff\" d=\"M34.46 41.07c-.35-1.14-.32-2 .12-2.56a2.32 2.32 0 0 1 1.73-.78c1.22-.04 1.7.82 2.4 3.07a46.9 46.9 0 0 0 1.8 4.6 44.64 44.64 0 0 0 2.06 3.87c.35.55.66.71 1.02.71.63 0 1.1-.35 1.1-.98 0-.36-.35-.95-.55-1.3-.35-.67-1.1-2.01-1.61-3.15a72.75 72.75 0 0 1-1.7-4.26c-.31-.83-.67-2-1.02-3.2-.55-1.8-.83-2.9-.83-3.74 0-1.26.95-2.08 2.37-2.08 1.02 0 1.69.5 2.12 2.36.35 1.7.98 4.45 1.89 6.62a57.31 57.31 0 0 0 1.97 4.26c.31.55.67 1.02.67 1.22 0 .27-.52.7-.95 1.3-.28.35-.4.59-.4.86 0 .24.16.48.4.8.24.3.47.62.75.62.2 0 .31-.08.43-.23a13.58 13.58 0 0 1 4.1-3.47 18.1 18.1 0 0 1 5.11-1.85c.71-.16.91-.32.91-.71 0-.48-.35-.75-.83-.8-.35-.03-.67 0-1.22.05-.39.04-.59-.16-.82-.67-.71-1.5-2.09-4.02-2.91-7.25a30.98 30.98 0 0 1-.87-4.73c-.12-.87.04-1.18.47-1.58a2.73 2.73 0 0 1 2.28-.47c.91.24 1.38.95 1.85 3.7a27.6 27.6 0 0 0 3.58 9.7c.63 1.1 1.42 2.25 2.25 3.35-.16.48-.44.79-1.34 1.54a10.1 10.1 0 0 0-2.64 3 6.77 6.77 0 0 0-.86 3.02c0 .75.15.9.63.9.82 0 1.5-.15 1.53-.5.2-1.42.43-2.33 1.22-3.4.47-.58 1.3-1.33 1.97-1.92a5.18 5.18 0 0 0 2.12-3.07c.2-.67.44-1.3.75-1.9.75-1.37 2.09-2.95 4.17-2.95.67 0 1.22.2 1.58.75.2.32.27.75.27 1.06 0 .71-.5 1.7-.74 2.17-.87 1.85-2.29 4.69-2.29 7.96 0 7.13-2.51 10.68-4.48 12.49a12.64 12.64 0 0 1-8.54 3.39c-2.56 0-5.2-.75-7.37-2.17-3.03-2-4.92-5.16-6.85-8.31a60.43 60.43 0 0 1-4.25-8.47 79.96 79.96 0 0 1-2.55-6.87z\"/><path fill=\"#fff\" d=\"M44.58 27.87c0-1.5 1.38-2.08 2.52-2.08 1.22 0 1.65.59 1.96 1.85.28 1.02.4 1.97.52 3.07.15 1.1.7 3.62.86 4.26.6 2.16 1.77 4.73 3.19 7.8-.83.27-1.93.74-2.68 1.14-.94.51-1.14.51-1.53-.24a32.9 32.9 0 0 1-1.93-4.02 29.14 29.14 0 0 1-1.22-3.82c-.59-2.48-.67-3.2-1.14-4.34a18.33 18.33 0 0 1-.55-3.62z\"/><path fill=\"#000\" d=\"M30.49 54.25c-.84.52-5.25 2.83-6.25 3.2-.71.27-1.49.2-1.94-1.14-.56-1.67.23-2.06 1.34-2.5.99-.38 5.08-1.86 6.04-2.12.67-.18 1.13.05 1.45.85.36.89.11 1.24-.64 1.71zm-3.64-9.38c-1.2.03-6.12-.18-7.25-.3-.98-.09-1.38-.52-1.27-1.94.11-1.53.7-1.62 1.67-1.53 1.12.1 6.28.95 7.1 1.2.86.27.98.66.87 1.4-.13.97-.57 1.15-1.12 1.17zm.6-10c-1.35-.43-5.39-2.87-6.44-3.5-.95-.58-1.07-1.1-.4-2.35.58-1.1 1.23-1.3 2.2-.8 1.11.58 5.07 3.66 5.82 4.27.73.59.63.9.25 1.65-.38.75-.74.95-1.42.73z\"/></svg>"
```

Comment:
I couldn't find a better icon.

Reply:
@Sergey-Lyapin how does it look like?

Reply:
@ameshkov  <details>
<summary>Details</summary>

![image](https://github.com/AdguardTeam/HostlistsRegistry/assets/8361299/c62433dd-58dc-48fe-bb1c-d293f5f3346d)


</details>

Reply:
Well, kind of okay